server/src/modules/uploads/upload.controller.ts

```
@@ -0,0 +1,46 @@
+import { NextFunction, Request, Response } from 'express';
+import { apiResponse } from '../../utils/response';
+import { DocItem } from './upload.model';
+import { createDocs, getDocs, purgeDocs } from './upload.service';
+
+async function uploadHandler(req: Request, res: Response) {
+  try {
+    const documentGroups: DocItem[] = [];
+
+    Object.keys(req.body)
+      .filter((item) => item.endsWith('_docTypeID'))
+      .forEach((key) => {
+        const prefix = key.split('_')[0];
+        const docItem: DocItem = {
+          docTypeID: req.body[`${prefix}_docTypeID`],
+          docNo: req.body[`${prefix}_docNo`],
+          description: req.body[`${prefix}_description`],
+          files: req.uploadedFiles.names[`${prefix}_files`],
+          urls: req.uploadedFiles.urls[`${prefix}_files`],
+        };
+        documentGroups.push(docItem);
+      });
+
+    const result = await createDocs(documentGroups);
+
+    if (result) apiResponse(res, 'Files uploaded successfully!');
+  } catch (error) {
+    console.log('save error', error);
+  }
+}
+
+async function getDocsHandler(req: Request, res: Response, next: NextFunction) {
+  try {
+    const result = await getDocs();
+    apiResponse(res, result, 201);
+  } catch (error) {
+    console.log('-- geDocs error', error);
```

firstly, you need to use the logger rather than `console.log`

secondly, you need to throw the error, for our global error handler to catch.